src/desktop/components/main_layout/header/index.styl

```diff
@@ -22,7 +22,6 @@
 
 #main-layout-header-left
   flex-grow 1
-  max-width 750px
```

Comment:
Wow, was this all that was needed to make the nav bar look like whats in the ticket? I tried to fix this for like an hour and couldn't figure it out.

Reply:
lol 🍃 landed in the CSS vortex